test/api_integration/apis/data_views/fields_for_wildcard_route/response.ts

```diff
@@ -249,11 +249,28 @@ export default function ({ getService }: FtrProviderContext) {
         .get(FIELDS_FOR_WILDCARD_PATH)
         .set(ELASTIC_HTTP_VERSION_HEADER, INITIAL_REST_VERSION_INTERNAL)
         .set(X_ELASTIC_INTERNAL_ORIGIN_REQUEST, 'kibana')
-        .query({ pattern: 'basic_index' })
+        .query({ pattern: 'basic_index', allow_no_index: true })
+        .expect((response) => {
+          if (response.statusCode !== 200) {
+            log.debug(response.body);
+          }
+        })
```

Comment:
Is this intentionally left in the test or was this just for debugging?

Reply:
I left it so we can see it logs if it ever fails again in the future.